src/main/java/demo/util/Pair.java

```diff
@@ -0,0 +1,4 @@
+package demo.util;
+
+public record Pair(String tableName, String idFieldName) {
```

Comment:
@romanovosad87 the name is not obvious when you see it in the code. How come `tableName` + `idFieldName` is called `Pair`? 😅